### DISCUSSION TITLE
Combo Box

### DISCUSSION THREAD

**Discussion by KamdemDarren** [TomSchimansky/CustomTkinter#132] [Ideas]
hello TomSchimansky , plz can you add comboBox design in the customCTKinter

**Comment by sumeshir26** [comment]
Its alreaday being worked on...

**Comment by TomSchimansky** [comment]
CTkComboBox and CTkOptionMenu are now available with version 4.3.0:
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkComboBox
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkOptionMenu

<img width="651" alt="Bildschirmfoto 2022-06-02 um 01 15 49" src="https://user-images.githubusercontent.com/66446067/171517978-42c048a8-7174-4d66-8771-1a852efccb7b.png">
